Makefile

```diff
@@ -6,6 +6,8 @@ all:
 	cd edge && $(MAKE)
 	cd keadm && $(MAKE)
 	cd edgesite && $(MAKE)
+	cd device/bluetooth_mapper && make bluetooth_mapper_image
```

Comment:
Can we keep it consistent with the above commands ?

Reply:
Updated